src/role-access/containers/specific-access/specific-access-approved/specific-access-approved.component.scss

```diff
@@ -0,0 +1,3 @@
+.govuk-grid-column-two-thirds {
+  padding-left: 0;
```

Comment:
I don't think you should be overriding the GDS standards. What is causing the padding?

Reply:
@paulhowes-HMCTS have removed original css rule added and used `govuk-!-padding-left-0` class as discussed